chromadb/test/segment/distributed/test_protobuf_translation.py

```diff
@@ -1,376 +0,0 @@
-import uuid
```

Comment:
should we remove all the protogen stuff too then?

Reply:
this PR is focused on just removing tests, but yes I'd like to